src/main/java/com/Acrobot/ChestShop/Listeners/PreShopCreation/PriceChecker.java

```diff
@@ -72,7 +72,7 @@ public static void onPreShopCreation(PreShopCreationEvent event) {
     }
 
     private static boolean isInvalid(String part) {
-        char characters[] = {'B', 'S'};
+        char[] characters = {'B', 'S'};
```

Comment:
This seems unnecessary.

Reply:
It's unnecessary but `char characters[]` is a C-style array declaration.

Reply:
Please just revert it, nothing else in the project does it like this and unnecessary changes should generally be avoided always to keep git blame easier to understand.

Reply:
Actually I misread the change here and I agree that it should be done like you did it now and it seems that it's not consistent in the whole code base but anyways, it has nothing to do with this change and does not belong into this commit. I prefer one-change, one-commit for such a complex code base so if you really want then feel free to open a separate PR fixing this issue and ideally others throughout the whole project.